src/jblockmain/Measurements.java

```diff
@@ -238,8 +238,13 @@ private void assignMeasurements(BufferedReader fileStream, boolean isTabbed)
 
                     for (int i = 0; i < numCustomMeasurements; ++i)
                     {
+                        if (idValues[i].equals("Unavailable"))
+                        {
+                            System.out.println("Measurement " + idName[i] + " is not available for user " + userNames.get(numUsers - 1));
+                            idValues[i] = "0.0";
+                        }
```

Comment:
Thought this was done on another issue  somewhere?

Reply:
I merged that issue into this issue to then merge it into development